clis/teliod/src/command_listener.rs

```diff
@@ -76,6 +76,19 @@ impl CommandListener {
                 })
                 .await
             }
+            ClientCmd::QuitDaemon =>
```

Comment:
IMHO it would be nice to log commands executed (e.g. `debug!()` or maybe `info!()`). This helps puts things into perspective when debugging various issues.

Obviously - we should log all commands not just `QuitDaemon`. Although it does not have to be this PR, it may be separate one

Reply:
Every command received by the daemon is logged [here](https://github.com/NordSecurity/libtelio/pull/1012/files#diff-c053616e6c57a08149244bd73aba33e23957c3eb3174c20b8ee7a02d2c95a593L254).